v4.6.1-releasenotes.txt

```diff
@@ -0,0 +1,108 @@
+BISDN Linux
+v4.6.1 Release Notes
+Date: 2022-06-08
+
+! New in this release
+- baseboxd: updated to version 1.11.5
+- Linux kernel updated to version 5.15.44
+- stfp and sftp-server were added in preparation of OpenSSH 9.0+ upgrades on host systems (which changes the default protocol of scp to SFTP)
+
+! Bug fixes
+- baseboxd: a fix was added to properly handle default routes on data ports
```

Comment:
We fixed the sfp leds for as4630-54pe.
We fixed the reported fan direction of the psus of as5385-54x.

Reply:
I could not find any commits regarding the latter issue about the PSUs.

Reply:
> as5835-54x: fix reporting of psu fan direction

Reply:
Mention of the SFP LED fix has been added

Reply:
Added Fan PSU direction